DataFormats/CSCDigi/interface/CSCCorrelatedLCTDigi.h

```diff
@@ -78,6 +79,7 @@ class CSCCorrelatedLCTDigi {
   int getBX() const { return bx; }
 
   /// return CLCT pattern number (in use again Feb 2011)
+  /// This function should not be used for Run-3
```

Comment:
In that case, IMO it would be better to replace the line below with something like this:
int getCLCTPattern() const { return ( (version_ == Version::0) ? (pattern & 0xF) : -1 ); }

Reply:
Yes, good point.